src/main/java/model/collection/list/CustomArrayList.java

```diff
@@ -0,0 +1,48 @@
+package model.collection.list;
+
+import entity.CustomList;
+import model.ticket.Ticket;
+
+import java.util.Arrays;
+
+public class CustomArrayList implements CustomList {
+    private Ticket[] myArray;
+    private int index = 0;
+
+    public CustomArrayList() {
+        myArray = new Ticket[5];
+    }
+
+    @Override
+    public void put(Ticket ticket) {
+        if (index >= myArray.length) {
+            myArray = Arrays.copyOf(myArray, myArray.length + 1);
+        }
+        myArray[index] = ticket;
+        index++;
+    }
+
+    @Override
+    public Ticket get(int index) {
+        if (index > myArray.length) {
+            throw new IndexOutOfBoundsException("Index " + index + " out of bounds for length " + myArray.length);
+        } else {
+            return myArray[index];
+        }
+    }
+
+    @Override
+    public void delete(int index) {
+        if (index >= 0 && index <= myArray.length - 1) {
```

Comment:
you have "if" but without else.
1. add else case
2. lor eave as it is and return true if element was deleted and false if it was not deleted instead of void

Reply:
Changed return type in delete method from void to boolean